source/code/source/playfab/QoS/PlayFabQoSApi.cpp

```diff
@@ -138,7 +138,7 @@ namespace PlayFab
             bool allTimedOut = true;
             for(auto& regionRes: result.regionResults)
             {
-                if(regionRes.latencyMs != timeoutMs)
+                if(regionRes.latencyMs != static_cast<int>(timeoutMs))
                 {
```

Comment:
what is this block of code checking exactly?  is there some special value for latencyMs that timeoutMs needs to match in order to be considered timed out?

Reply:
If we time out, regionRes.latencyMs is expected to have hit the timeout that users have set elsewhere (Passed in as a parameter). Rather than change the signature elsewhere (which I think may be a public facing API), I silenced a warning by adding this static cast.